app/__main__.py

```diff
@@ -269,13 +269,13 @@ def log_app_debug_info_and_force_kill_after_delay():
         logger.error('ClusterRunner did not exit within {} seconds. App debug info:\n\n{}.',
                      seconds, app_info.get_app_info_string())
         logger.critical('ClusterRunner seems to be hanging unexpectedly. Hard killing the process. Farewell!')
-        os._exit(1)
+        os._exit(1)  # pylint: disable=protected-access
 
     # Execute on a daemon thread so that the countdown itself will not prevent the app from exiting naturally.
     threading.Thread(target=log_app_debug_info_and_force_kill_after_delay, name='SuicideThread', daemon=True).start()
 
 
-def main(args):
+def main(args=sys.argv[1:]):
```

Comment:
Up to you, but I think maybe `args=None` will do the same thing (if you wanted to leave this as-is on line 303 below). If I remember correctly `argv[1:]` is the default in argparse internals anyway. I think the only reason for accepting args in the main method was to make testing cleaner.

Reply:
Ah, good point. I'll check if None works.